AK/UFixedBigInt.h

```diff
@@ -314,6 +313,11 @@ requires(sizeof(T) >= sizeof(u64) && IsUnsigned<T>) class UFixedBigInt {
         return *this;
     }
 
+    static constexpr size_t my_size()
```

Comment:
`consteval` this?

Reply:
I tried it with `consteval`, but it produced the same issue as with the bare `sizeof`. Probably a compiler bug

Reply:
technically sizeof(R) is sizeof(T)*2, this might help?
otherwise, does moving the requires clause to after the function definition help?

Reply:
> technically sizeof(R) is sizeof(T)*2, this might help?
I think that would make this code less readable; but definitely could work too

>  otherwise, does moving the requires clause to after the function definition help?
Still, invalid application of sizeof() to an incomplete type"